frontend/app/routes/_gcweb-app.personal-information.phone-number.edit.tsx

```diff
@@ -10,8 +10,7 @@ import { sessionService } from '~/services/session-service.server';
 import { userService } from '~/services/user-service.server';
 import { getTypedI18nNamespaces } from '~/utils/locale-utils';
 
-const i18nNamespaces = getTypedI18nNamespaces('update-phone-number');
-
+const i18nNamespaces = getTypedI18nNamespaces('personal-information', 'gcweb');
```

Comment:
Remove `gcweb` ns.

Reply:
yeah i had a merge conflict and re-added (not on purpose). Should be removed now.